<ui version="4.0" >
 <class>VideoSettings</class>
 <widget class="QWidget" name="VideoSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer_10" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="webcamPreviewLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>176</width>
         <height>144</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>198</width>
         <height>162</height>
        </size>
       </property>
       <property name="styleSheet" >
        <string notr="true" >background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>选择视频设备:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="webcamDeviceComboBox" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>发送的视频大小</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="videosizecomboBox" >
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text" >
         <string>352 X 288</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>704 X 576</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1280 X 720</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>1920 X 1080</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" >
     <item>
      <widget class="QLabel" name="label_65" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>高清转换器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="crossbarlistcomboBox" >
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" >
     <item>
      <widget class="QLabel" name="label_66" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>高清输入类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="crossbarclasscomboBox" >
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" >
     <item>
      <widget class="QLabel" name="label_67" >
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>发送图像质量   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="videoQualitySlider" >
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>4</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval" >
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11" >
     <item>
      <widget class="QPushButton" name="webcamPreviewButton" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>预览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_68" >
       <property name="text" >
        <string>     普通     好   很好  非常好</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12" >
     <item>
      <widget class="QCheckBox" name="enableVideoCheckBox" >
       <property name="text" >
        <string>Enable video</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="makeTestVideoCallButton" >
       <property name="text" >
        <string>Make a test video call</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_8" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13" >
     <item>
      <spacer name="horizontalSpacer_15" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton" >
       <property name="text" >
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
